packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -600,4 +624,22 @@ class _DragAvatar<T> extends Drag {
       )
     );
   }
+
+  Velocity _restrictVelocityAxis(Velocity velocity) {
+    if (axis == null) {
+      return velocity;
+    }
+    return new Velocity(
+        pixelsPerSecond: _restrictAxis(velocity.pixelsPerSecond));
```

Comment:
put this all on one line or put a comma after the first ) so that the next one ends up on the next line.

Reply:
Done